src/ui/component/button/view.jsx

```diff
@@ -20,7 +20,6 @@ type Props = {
   description: ?string,
   type: string,
   button: ?string, // primary, secondary, alt, link
-  noPadding: ?boolean, // to remove padding and allow circular buttons
```

Comment:
Nice. We should kill properties that are style only like this whenever possible.